.gitignore

```diff
@@ -6,6 +6,7 @@ src/core_*/*.f90
 src/framework/*.f90
 src/driver/*.f90
 src/operators/*.f90
+src/core_atmosphere/physics/physics_mmm
```

Comment:
The `physics_mmm` directory is already ignored in the `hotfix-v8.2.1` branch in the `src/core_atmosphere/physics/.gitignore` file.